docs/install/project.md

```diff
@@ -20,11 +20,11 @@ terminal.
 * launch VScode, either through your application menu or by typing
   `code`
 
-* On the main screen, or in the File menu, click "Open folder", and
+* On the main screen, or in the File menu, click "Open folder" (on a Mac, just "Open"), and
```

Comment:
There is no need for this parenthesis, this is the whole point of having separate installation instructions files. We want to talk to one user at a time. Simply write "click 'Open'".

Reply:
This file is shared between all OSes.

Reply:
Sorry, I missed the file change while reading GitHub diff.